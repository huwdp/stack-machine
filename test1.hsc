push 10
push 2
sub
push 3
mul
push 2
div
printLine
