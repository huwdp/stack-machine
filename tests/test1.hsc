pushi 10
pushi 2
subi
pushi 3
muli
pushi 2
divi
printl
