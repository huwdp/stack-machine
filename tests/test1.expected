Welcome to Huw's Stack Machine
12
